packages/react/src/side-nav/SideNavLink.tsx

```diff
@@ -1,118 +1,108 @@
-				paddingRight={1}
-				borderBottom
-				borderBottomWidth="sm"
-				borderColor="muted"
-				focusRingFor="keyboard"
-				{...props}
-				css={{
-					textDecoration: 'none',
-					paddingLeft: `${depth}rem`,
+		<Flex
+			as={Link}
+			aria-current={isCurrentPage ? 'page' : undefined}
+			gap={0.75}
+			color="muted"
+			fontSize="xs"
+			paddingTop={1}
+			paddingBottom={depth > 1 ? 1 : 0.75}
+			paddingRight={1}
+			borderBottom
+			borderBottomWidth="sm"
+			borderColor="muted"
+			focusRingFor="keyboard"
+			{...props}
+			css={{
+				textDecoration: 'none',
+				paddingLeft: `${depth}rem`,
 
-					'&:hover': {
-						color: boxPalette.foregroundText,
-						backgroundColor: collapsingSideBarLocalPalette.hover,
-						'& span:last-of-type': packs.underline,
-					},
+				'&:hover': {
+					color: boxPalette.foregroundText,
+					backgroundColor: collapsingSideBarLocalPalette.hover,
+					'& span:last-of-type': packs.underline,
+				},
 
-					...(isCurrentPage && {
-						position: 'relative',
-						color: boxPalette.foregroundText,
-						backgroundColor: collapsingSideBarLocalPalette.hover,
-						fontWeight: tokens.fontWeight.bold,
-						':before': {
-							content: '""',
-							position: 'absolute',
-							top: 0,
-							left: 0,
-							bottom: 0,
-							borderLeftWidth: tokens.borderWidth.xl,
-							borderLeftStyle: 'solid',
-							borderLeftColor: boxPalette.selected,
-							pointerEvents: 'none',
-						},
-					}),
-				}}
-			>
-				<SideNavLinkIndicator depth={depth} />
-				<span>{label}</span>
-			</Flex>
-			{children}
-		</SideNavListItem>
-	);
-}
+				...(isCurrentPage && {
+					position: 'relative',
+					color: boxPalette.foregroundText,
+					backgroundColor: collapsingSideBarLocalPalette.hover,
+					fontWeight: tokens.fontWeight.bold,
+					':before': {
+						content: '""',
+						position: 'absolute',
+						top: 0,
+						left: 0,
+						bottom: 0,
+						borderLeftWidth: tokens.borderWidth.xl,
+						borderLeftStyle: 'solid',
+						borderLeftColor: boxPalette.selected,
+						pointerEvents: 'none',
+					},
+				}),
+			}}
+		>
+			<SideNavActiveAncestorIndicator depth={depth} />
 
-type SideNavListItemProps = PropsWithChildren<{
-	depth: number;
-	isActive?: boolean;
-}>;
+			<span css={{ flexGrow: 1 }}>{label}</span>
 
-function SideNavListItem({ children, depth, isActive }: SideNavListItemProps) {
-	return (
-		<li
-			css={
-				depth === 1 && isActive
-					? {
-							position: 'relative',
-							':before': {
-								content: '""',
-								position: 'absolute',
-								top: 0,
-								left: 0,
-								bottom: 0,
-								borderLeftWidth: tokens.borderWidth.xl,
-								borderLeftStyle: 'solid',
-								borderLeftColor: boxPalette.borderMuted,
-								pointerEvents: 'none',
-							},
-					  }
-					: undefined
-			}
-		>
-			{children}
-		</li>
+			{hasNestedItemsIndicator &&
+				(isOpen ? (
+					<ChevronDownIcon
+						aria-hidden={false}
+						aria-label=". Sub-level links below."
```

Comment:
Should we count here too? Should we just use the same "Has"?

Reply:
I don't think so because the number of list items will get announced but I do think we should conditionally pluralise. What do you think?

Reply:
>Should we just use the same "Has"

Maybe. I really want to align the announcement with the difference the chevrons convey. Partly to offer a match to sighted screen reader users and partly because we intentionally do this chevron rotation, so the UX is considered valuable. And we also cause we _can_ 🙌

Reply:
They count will only get announced when you're in 1 of the sub-levels, but seeing as though you probably heard how many there were when you clicked the original link, I think it's fine.

Perhaps just pluralise then

Reply:
- [ ] conditional pluralise